test/network.jest.js

```diff
@@ -133,13 +133,28 @@ describe('Request.postData', function() {
     expect(request).toBeTruthy();
     expect(request.postData()).toBe('{"foo":"bar"}');
   });
+  it('should work with binary', async({page, server}) => {
+    await page.goto(server.EMPTY_PAGE);
+    server.setRoute('/post', (req, res) => res.end());
+    let request = null;
+    await page.route('/post', route => route.continue());
```

Comment:
Why intercept?

Reply:
We update postData upon interception and it can rewrite out value...